src/usersyncroniser.ts

```diff
@@ -218,6 +225,11 @@ export class UserSyncroniser {
         const guildState = Object.assign({}, DEFAULT_GUILD_STATE, {
             id: newMember.id,
             mxUserId: `@_discord_${newMember.id}:${this.config.bridge.domain}`,
+            roles: newMember.roles.map((role) => { return {
```

Comment:
This can be shortened to 
```
newMember.roles.map((role) => {
                name: role.name,
                color: role.color,
                position: role.position,
})
```

I believe

Reply:
Doesn't seem so, unfortunately (I just tried)

Reply:
Really, oh well. I thought that worked :|